### PR TITLE
change to ingestion by class on NIST, CWE, and ARCOS data

### PR DESCRIPTION
**Pull request by kityansiu**

### CODE REVIEW

File: manifests/arcos.yaml [hunk 1]
```diff
@@ -11,24 +11,19 @@ steps:
...
 
   - model: ../STR-Ontology/OwlModels/import.yaml
-  - nodegroups: ../nodegroups/ingestion/arcos.arbiter
 
   - model: ../RTX-Ontology/OwlModels/import.yaml
 
   - data: ../RACK-Ontology/ontology/NIST-800-53/import.yaml
 
-  - model: ../RACK-Ontology/OwlModels/MITRE-CWE.yaml
+  - model: ../RACK-Ontology//OwlModels/MITRE-CWE.yaml
```
Comment (Ptival) [inline manifests/arcos.yaml:27]: This looks unnecessary.

**Review by Ptival — COMMENTED** [review]
All classes look correct.

What is the intent behind the removal of the nodegroups in `arcos.yaml`?

File: manifests/arcos.yaml (hunk 1 above)
Comment (kityansiu) [reply manifests/arcos.yaml:27]: yikes. typo.

Comment (kityansiu) [reply manifests/arcos.yaml:27]: I just wanted to make it lighter weight, so we don't have to load all these nodegroups into RACK-in-a-Box if we are not relying on them for ingestion.

**Pull request merged**